cuda/matrix/csr_kernels.cu

```diff
@@ -718,6 +727,8 @@ void spmv(std::shared_ptr<const CudaExecutor> exec,
                 cusparseSetPointerMode(handle, CUSPARSE_POINTER_MODE_DEVICE));
 
             cusparse::destroy(descr);
+        } else {
+            GKO_NOT_IMPLEMENTED;
         }
     }
```

Comment:
I think we actually also need a
```c++
else {
    GKO_NOT_IMPLEMENTED;
}
```
here in case the `a->get_strategy()` is not known.

Reply:
done

Reply:
Thanks!